### ISSUE TITLE
Support scraping a Facebook profile's uploaded and tagged-in photos

### ISSUE THREAD

**Issue by JustAnotherArchivist** [JustAnotherArchivist/snscrape#24]

**Comment by JustAnotherArchivist** [comment]
Is there actually any way to access tagged photos without logging in? I couldn't find anything. UFS (before it went closed-source) as well as the other similar tools I saw all require an account.

**Comment by github-userx** [comment]
I truly don’t think so. Even with login in , they made it much more difficult ever since removing graph-search features

**Comment by JustAnotherArchivist** [comment]
Right. Well, then that won't be happening in snscrape, unfortunately. I'll turn this issue into just uploaded photos.

**Comment by thebham24** [comment]
is there a way to get the photos attached to a single fb post in a group? I could find only links to the first 5 images. all images after the 6th were to be browsed via opening the carousel and clicking next. My idea is to get create a single pdf file for a post with all information of that post. i use expandall plugin to expand all post comments. and then create a pdf with the information on screen. but the only issue is a getting the images in that post which are more than 6. (as fb shows preview of only the first 5)

**Comment by JustAnotherArchivist** [comment]
@thebham24 That's not what this issue is about. Please file a separate one and include examples there.
